pkg/solana/contract_cache.go

```diff
@@ -111,6 +115,14 @@ func (c *ContractTracker) PollState() {
 				err := c.fetchState(ctx)
 				if err != nil {
 					c.lggr.Errorf("error in PollState.fetchState %s", err)
+				} else {
+					// We have successfully read state from the contract.
+					// Signal that we are ready to start libocr.
```

Comment:
One thing to note is that, the transmission fetch needs to be able to handle empty transmissions without erroring regardless, as we can't produce transmissions until libocr is started so there will inevitably be a period of time where that is empty on first contract deploy